extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/JpaJandexScavenger.java

```diff
@@ -196,13 +213,29 @@ private static void addClassHierarchyToReflectiveList(IndexView index, JpaEntiti
             }
         }
 
-        //Capture this one (for various needs: Reflective access enablement, Hibernate enhancement, JPA Template)
+        //Capture this one (for various needs: Reflective access enabling, Hibernate enhancement, JPA Template)
```

Comment:
That was correct. It should be reverted.

Reply:
Done.